src/mixins/links.css

```diff
@@ -0,0 +1,18 @@
+@import './focusable.css';
+
+a {
```

Comment:
This is in a mixin file because I expect we may have other components with links in the shadow DOM, and we can just `@import` this when we get there.